### ISSUE TITLE
Little feature suggestion. I think most will agree.

### ISSUE THREAD

**Issue by CrowBBGN** [Chisel-2/Chisel-2#404]
This is something that I'm surprised isn't already in there...

So you know when you are going along and whacking all the blocks with the chisel. 
Me personally I love laying down lots of cobble and going at it with the chisel to get those
nice cobble bricks.. BUT you accidentally hit a block one too many times... Now you have to keep
whacking at it a dozen or more times until you cycle through and get back to the one you wanted.

Why not add it so when you hold Shift and hit the block you go backwards through the blocks.
Hit one too many? Hold shift, hit the block and there's your desired block.
I know many people get frustrated by hitting a block one too many times and having to cycle through
them all again. This would fix that and put a smile back on all our click happy faces.

Thanks for considering.

**Comment by TehNut** [comment]
Just put the block you want it to turn into in the chisel

**Comment by CrowBBGN** [comment]
I am very much aware of this but that would add more steps to what I am trying to accomplish.

Oops, hit one too many.
1, select pick.
2, break block.
3, pick up block,
4, select chisel.
5. open chisel.
6, put block in chisel.
7, select desired block.
8, put block in inventory.
9. close chisel gui
10, select block,
11, place block..... Done..

OR.......

Oops, hit one too many.
1, Hold shift
2, hit block. .... Done...

Which sounds better?

**Comment by TehNut** [comment]
No, I mean when you're running around smacking the blocks, stick one of whatever type you want in the chisel. It'll turn all the blocks you hit into that.

**Comment by CrowBBGN** [comment]
I appreciate you trying to help but I'm not reporting this as an issue or asking for help, just a feature request. Some people only want to change only one block so they hit it with the chisel and can accidentally hit it to many times. What I am personally doing is placing all cobble and hitting them so i get the nice cobble bricks. Some hit twice which for what I am doing is good since the next one is the busted cobble bricks. it gives the room a nice look. most are good some are falling apart, looks natural. but sometimes I hit one too many times and get the checker board looking one which I don't want. So a shift click would be really nice to have in my case and the single block cases.

**Comment by SgtPunishment** [comment]
What @TehNut is saying @CrowBBGN is you get the block of your choice and place that in the chisel then close the chisel gui, then when you hit the same block type (in your example cobble stone) it will turn the blocks you hit into the block you've selected by putting said block INTO the chisel gui

**Comment by CptSpaceToaster** [comment]
I think @CrowBBGN wanted to just speckle random cobble patterns everywhere.  So it's not that he CAN'T cram a block into the chisel and hammer away...  It's that there really isn't a good way for Crow to select 1 of these 3 patterns of cobble randomly as they chisel away over a large area.

**Comment by TheCricket** [comment]
its easy enough to do

**Comment by DoomRater** [comment]
Though personally why not just use multiple chisels and occasionally whack with the second or third pattern?  The only reason I wouldn't do this myself is because I hate having multiple damaged tools (and haven't seen if there's a way to configure the mod so that the chisels never take damage).
